lucene/core/src/test/org/apache/lucene/index/TestNumericDocValuesUpdates.java

```diff
@@ -1483,6 +1484,83 @@ public void testAddIndexes() throws Exception {
     IOUtils.close(dir1, dir2);
   }
 
+  public void testUpdatesAfterAddIndexes() throws Exception {
```

Comment:
It might be better to split into one unit test for the addIndexes case when the updated field already exists in the segment, and another one for the case when the updated field doesn't exist in the segment yet?

Reply:
++, I pushed https://github.com/apache/lucene-solr/pull/1394/commits/00d755888d4744ff6a1beba17e20580f43841f12